src/components/Form.js

```diff
@@ -44,7 +45,9 @@ function Form() {
           onChange={(e) => setauthor(e.target.value)}
           required
         />
-        <button type="submit" onClick={handelAddBookClick}>Add Book</button>
+        <button type="submit" className="add-btn" onClick={handelAddBookClick}>
+          Add Book
+        </button>
       </form>
     </div>
```

Comment:
> - [ ] Please make sure to adhere closely to the design guidelines whenever possible. This is important because it maintains consistency and ensures a unified visual experience throughout the project.👍
> 
> Provided Design:
> 
> ![add-1](https://github.com/MostafaShahat2021/Bookstore/assets/40009354/0da5760a-66f3-4796-909a-3ec936db419d)
> 
> Your design:
> 
> ![add-1-2](https://github.com/MostafaShahat2021/Bookstore/assets/40009354/78a73ca8-23c7-40a0-acfb-07c6ef273afe)
>

Reply:
Hi @Johnadibe I think we don't need that because it is mentioned in the project requirements to replace the category with the author.
![image](https://github.com/MostafaShahat2021/Bookstore/assets/79546797/e318d9a7-adcc-4991-80d4-7c8e8919e82a)
[link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#project-requirements)

Reply:
well resolved! ✔️